pallets/manta-sbt/src/mock.rs

```diff
@@ -0,0 +1,322 @@
+            metadata.name,
+            metadata.symbol,
+            metadata.decimals,
+            metadata.is_frozen,
+        )
+    }
+
+    fn update_asset_metadata(
+        asset_id: &StandardAssetId,
+        metadata: AssetStorageMetadata,
+    ) -> DispatchResult {
+        Assets::force_set_metadata(
+            Origin::root(),
+            *asset_id,
+            metadata.name,
+            metadata.symbol,
+            metadata.decimals,
+            metadata.is_frozen,
+        )
+    }
+}
+
+parameter_types! {
+    pub const DummyAssetId: StandardAssetId = 0;
+    pub const NativeAssetId: StandardAssetId = 1;
+    pub const StartNonNativeAssetId: StandardAssetId = 8;
+    pub NativeAssetLocation: AssetLocation = AssetLocation(
+        VersionedMultiLocation::V1(MultiLocation::new(1, X1(Parachain(1024)))));
+    pub NativeAssetMetadata: AssetRegistryMetadata<Balance> = AssetRegistryMetadata {
+        metadata: AssetStorageMetadata {
+            name: b"Dolphin".to_vec(),
+            symbol: b"DOL".to_vec(),
+            decimals: 18,
+            is_frozen: false,
+        },
+        min_balance: 1u128,
+        is_sufficient: true,
+    };
+    pub const AssetManagerPalletId: PalletId = ASSET_MANAGER_PALLET_ID;
+}
+
+/// AssetConfig implementations for this runtime
+#[derive(Clone, Eq, PartialEq)]
+pub struct MantaAssetConfig;
+impl LocationType for MantaAssetConfig {
+    type Location = AssetLocation;
+}
+impl AssetIdType for MantaAssetConfig {
+    type AssetId = StandardAssetId;
+}
+impl BalanceType for MantaAssetConfig {
+    type Balance = Balance;
+}
+impl AssetConfig<Test> for MantaAssetConfig {
+    type NativeAssetId = NativeAssetId;
+    type AssetRegistryMetadata = AssetRegistryMetadata<Balance>;
+    type StartNonNativeAssetId = StartNonNativeAssetId;
+    type NativeAssetLocation = NativeAssetLocation;
+    type NativeAssetMetadata = NativeAssetMetadata;
+    type AssetRegistry = MantaAssetRegistry;
+    type StorageMetadata = AssetStorageMetadata;
+    type FungibleLedger = NativeAndNonNative<Test, MantaAssetConfig, Balances, Assets>;
+}
+
+impl pallet_asset_manager::Config for Test {
+    type Event = Event;
+    type AssetId = StandardAssetId;
+    type Balance = Balance;
+    type Location = AssetLocation;
+    type AssetConfig = MantaAssetConfig;
+    type ModifierOrigin = EnsureRoot<AccountId32>;
+    type PalletId = AssetManagerPalletId;
+    type WeightInfo = ();
+}
+
+pub struct MantaPaySuspensionManager;
+impl pallet_manta_pay::SuspendMantaPay for MantaPaySuspensionManager {
+    fn suspend_manta_pay_execution() {
+        let _ = TransactionPause::pause_transactions(
+            RawOrigin::Root.into(),
+            vec![(
+                b"MantaPay".to_vec(),
+                vec![
+                    b"to_private".to_vec(),
+                    b"private_transfer".to_vec(),
+                    b"to_public".to_vec(),
+                ],
+            )],
+        );
+    }
+}
```

Comment:
Should we add a suspender for the SBT pallet as well ? Separate issue though.

Reply:
We could, less important than manta pay tho, as there are no funds to be drained when Ledger get borked

Reply:
Actually is the suspender even a good idea at this point, as MantaPay is not the only ledger on the blockchain.

Reply:
suspender would only stop MantaPay, I think it still make sense